annotation-model/scripts/JSONtest.js

```diff
@@ -306,6 +325,13 @@ JSONtest.prototype = {
       compareWith = 'and';
     }
 
+    var typeMap = {
+      'must'   : "",
```

Comment:
@halindrome guessing MUST's don't need a prefix because they actually are errors in this case?